searx/engines/presearch.py

```diff
@@ -128,7 +128,7 @@ def _get_request_id(query, params):
         # performs an IP-based geolocation of the user, we don't want that in
         # SearXNG ;-)
 
-        if l.territory:
```

Comment:
If we are adding `if l.territory`, do we also need to include `if l.language`?

Reply:
I don't think so .. SXNG only knows tags like `de-DE` or `de` .. but there is no *region-only* tag like `?-DE` .. so we always have a `l.language` when locale is not `None`.

Reply:
Makes sense to me. If it becomes a problem for any reason, it can be dealt with then.